<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resourceManagement</class>
 <widget class="QWidget" name="resourceManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>566</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="resourceSearch">
         <property name="toolTip">
          <string>Enter the search term</string>
         </property>
         <property name="whatsThis">
          <string>Use this text box to enter some keyword for the resource you want to find.</string>
         </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeResults">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupDetails">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Details</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="rightMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formDetails"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupOwner">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Owner</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formOwner"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget2">
       <layout class="QHBoxLayout" name="resourceCalender">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
